lib/src/license.dart

```diff
@@ -50,104 +51,22 @@ Future<LicenseFile> detectLicenseInFile(File file,
   return license ?? LicenseFile(relativePath, LicenseNames.unknown);
 }
 
+/// Returns an instance of [LicenseFile] if [originalContent] has contains a license(s)
+/// present in the [SPDX-corpus][1].
+///
+/// [1]: https://spdx.org/licenses/
 LicenseFile? detectLicenseInContent(String originalContent,
     {required String relativePath}) {
   var content = originalContent;
-  if (content.startsWith('// ')) {
-    content = content.split('\n').map((s) {
-      return s.startsWith('// ') ? s.substring(3) : s;
-    }).join('\n');
-  }
-  var stripped = _longTextPrepare(content);
-
-  String? version;
-  var versionMatch = _version.firstMatch(stripped);
-  if (versionMatch != null) {
-    version = versionMatch.group(1);
-    if (version!.isNotEmpty && !version.contains('.')) {
-      version += '.0';
-    }
-  }
+  final licenseResult = detectLicense(content, 0.85);
```

Comment:
Let's just set the confidence at 0.95

Reply:
Let's try to count number of licenses detect with 0.9 vs 0.95 (ideally, after we add back `_removeOmitableText`